lnbits/core/crud.py

```diff
@@ -69,6 +69,13 @@ async def get_user(user_id: str, conn: Optional[Connection] = None) -> Optional[
     )
 
 
+async def get_all_users() -> List[User]:
+    users = await db.fetchall("SELECT id FROM accounts")
+    assert users
```

Comment:
- I think `fetchall` returns an empty list if noting found. So this will always be true

Reply:
fixed